<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="contenedor" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <layout class="QGridLayout" name="tablero"/>
       </item>
       <item>
        <widget class="QWidget" name="stats" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="labelBlack">
            <property name="styleSheet">
             <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                            stop:0 #8B4513, stop:0.3 #7A3F1E, stop:0.6 #8B4513, stop:1 #5C3317);
border: 2px solid #4B2E1E;
border-radius: 10px;


</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelWhite">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                  stop:0 #d2b48c, stop:1 #deb887);  /* Gradiente madera suave */
border: 2px solid #a0522d;  /* Borde marr√≥n oscuro */
border-radius: 10px;


</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJuego">
    <property name="title">
     <string>Game Options</string>
    </property>
    <addaction name="actionMenu"/>
    <addaction name="actionNew_Game"/>
   </widget>
   <addaction name="menuJuego"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMenu">
   <property name="text">
    <string>Menu</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
